chroma_api/stratagem.py

```diff
@@ -63,6 +63,12 @@ def check_duration(duration_key, bundle):
             ),
         }
 
+    if duration < 0:
+        return {
+            "code": "{}_too_small".format(duration_key),
```

Comment:
The way this is currently designed means you also need to add new variants to this enum:

https://github.com/whamcloud/integrated-manager-for-lustre/blob/465e8fb076d3ac083d6f1dc4c4ec3a1c5d1cd402/iml-manager-cli/src/manager_cli_error.rs#L12-L29